packages/suite-web/e2e/tests/coinmarket/exchange.test.ts

```diff
@@ -1,5 +1,6 @@
 // @group_coinmarket
 
+// TODO: rewrite
```

Comment:
Do we have issue for this? I do not like skipped tests. Whats the plan to fix it?

Reply:
It will be created after the whole redesign.